plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -93,25 +95,66 @@
 {
     private static final Logger log = Logger.get(BaseJdbcClient.class);
 
+    public static final Type TRINO_PAGE_SINK_ID_COLUMN_TYPE = BigintType.BIGINT;
+
+    public static String getPageSinkIdColumnName(List<String> otherColumnNames)
```

Comment:
make private and move to bottom of the file

Reply:
ok - it also looks like i can make `TRINO_PAGE_SINK_ID_COLUMN_TYPE` package private - any objection?